gnmidiff/json.go

```diff
@@ -55,20 +56,23 @@ import (
 //   "openconfig-network-instance:config/type": "openconfig-network-instance-types:L3VRF",
 //   "openconfig-network-instance:name": "RED",
 // }
-func flattenOCJSON(json7951 []byte) (map[string]interface{}, error) {
-	// TODO: Add option to remove the namespace on paths of returned updates.
+//
+// When keepNamespace=false, then any namespace is removed from the flattened
+// *paths*, but still kept in any enum values.
```

Comment:
(nit) identity values, since enumerations don't have this namespacing.

Reply:
Done.